fluent_pose_synthesis/core/training.py

```diff
@@ -282,25 +242,19 @@ def export_samples(self, pose_output: Tensor, save_path: str, prefix: str):
         """
         for i in range(pose_output.shape[0]):
             pose_array = pose_output[i].cpu().numpy()  # (time, 1, keypoints, 3)
-            # time, people, keypoints, D = pose_array.shape
+            time, people, keypoints, dim = pose_array.shape
 
-            # Velocity-based confidence estimation
-            # Compute frame-to-frame differences: (time-1, people, keypoints, D)
-            diffs = np.diff(pose_array, axis=0)
-            # Compute L2 norm of motion per keypoint: (time-1, people, keypoints)
-            velocity = np.linalg.norm(diffs, axis=-1)
-            # Pad the first frame with zeros to restore time: (time, people, keypoints)
-            velocity = np.pad(velocity, ((1, 0), (0, 0), (0, 0)))
-
-            # Convert to confidence using exponential decay
-            confidence = np.exp(-velocity)
-            # Clip values to avoid exact 0 or 1
-            confidence = np.clip(confidence, 0.01, 1.0).astype(np.float32)
+            # Set confidence to all 1.0 for all keypoints
+            confidence = np.ones((time, people, keypoints), dtype=np.float32)
 
             # Wrap and export the pose data
             pose_body = NumPyPoseBody(fps=25, data=pose_array, confidence=confidence)
             pose_obj = Pose(self.pose_header, pose_body)
 
+            # Normalize for visualization
+            pose_obj.normalize(scale_factor=70)
```

Comment:
i think that here, for vis, it is best to use this:
https://github.com/sign-language-processing/pose/blob/master/src/python/pose_format/utils/generic.py#L57

so:
```py
normalize_pose_size(pose_obj)
```